### ISSUE TITLE
GUI lacks "log out" button

### ISSUE THREAD

**Issue by longears** [keybase/client#5861]
The OSX client has no way to log out or switch accounts, either from the main app or from the status bar interface.

"keybase logout" from the command line works.

**Comment by jeremyBanks** [comment]
Yeah... I just installed this on Windows, where I'm not familiar with the CLI, and signed in to the wrong account. I can't figured out how to sign out.

I'll probably end-up deauthenticating the device from another device and hoping that has the right effect, but not having a clear way to log out/deauthenticate on-device is a pretty major omission.

**Comment by cjb** [comment]
@jeremyBanks Ah, the "keybase logout" command line mentioned works from the "Keybase Shell" Windows start menu item.

**Comment by arunsathiya** [comment]
I don't think this is the case anymore. I see a `Sign out` button on the settings page of the Mac app - OS X client. Hopefully that is what you were referring to as well, @longears?

![screen shot 2018-08-14 at 09 56 11](https://user-images.githubusercontent.com/18581859/44071693-65ef73be-9fa8-11e8-82cd-20e526cb873b.png)

**Comment by todb-r7** [comment]
[This response](https://github.com/keybase/client/issues/5861#issuecomment-412751198) is now the right answer, so this issue can be closed, @keybase. IMO.
